package.json

```diff
@@ -1,22 +1,31 @@
 {
-  "name": "@squadcraft/eslint-config",
+  "name": "@squadcraft/eslint-config-squadcraft",
   "version": "1.0.0",
```

Comment:
1.0.0-rc if u want to publish it and check only, 1.0.0 if we will publish it after all without testing :)

Reply:
`rc` stands for `release candidate`

Reply:
How are we going to test our package. Is that even possible? The only thing that comes to my mind is checking:
- eslint's and airbnb's packages deprecation and their compability
- rules deprecation
There is not any coded functionality that needs to be tested. 
Imo we can leave version 1.0.0 as it is, and publish our package when it's ready to use :)

Reply:
to make eslint and airbnb packages unchanged just remove `^` in package.json next to them.
Also - version can be 1.0.0-rc to test it in main repo - that's how you test packages.
You can also use npm link but it depends how you like it.